ckan/lib/uploader.py

```diff
@@ -96,6 +98,36 @@ def get_max_resource_size() -> int:
     return config.get('ckan.max_resource_size')
 
 
+def _file_hashnlength(local_path: str) -> tuple[str, int]:
+    BLOCKSIZE = 65536
```

Comment:
It'd be nice to have a comment on why this block size was chosen. According to https://stackoverflow.com/a/17751367 this is the ideal block size for larger files

Reply:
It's copied from [ckanext-archiver](https://github.com/qld-gov-au/ckanext-archiver/blob/3820e618ce479e13268aa56196aef0e3cf79eae0/ckanext/archiver/tasks.py#L538), actually :D

The commit that originally added it, six years ago, has little information, unfortunately. https://github.com/qld-gov-au/ckanext-archiver/commit/9670599ca10fe63f173f2751d67c197ca93a8034